lib/screen/github_doc.dart

```diff
@@ -106,15 +108,36 @@ class _GithubDocPageState extends State<GithubDocPage> {
     }
   }
 
-  Future<String> _githubPathForDocument(String docFileName) async {
-    var prefix = widget.payload["prefix"] ?? '';
+  String _githubPath() {
+    final prefix = widget.payload.prefix;
+    final document = widget.payload.document;
+    final language =
+        widget.payload.fileNameAsLanguage ? '/${_getLanguage()}.md' : '';
 
-    if (prefix.isEmpty) {
-      prefix = (await isAppCenterBuild() || kDebugMode)
-          ? "/bitmark-inc/autonomy-apps/develop/docs/"
-          : "/bitmark-inc/autonomy-apps/main/docs/";
-    }
+    return prefix + document + language;
```

Comment:
.md is missing for fileNameAsLanguage = false?

Reply:
it is supposed to have .md in `document` if fileNameAsLanguage = false